src/components/tab/Tab.jsx

```diff
@@ -5,6 +5,7 @@ import { styles } from '~/components/tab/Tab.styles'
 const Tab = ({ activeTab, onClick, children }) => {
   return (
     <Button
+      aria-selected={activeTab}
```

Comment:
Why it was added in PR with tests?

Reply:
To control whether the Tab is active if the activeTab prop is true.
Should it be done in a different way? 

![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/93f62d1c-ea8f-488f-a500-0522e923d712)

Reply:
So, if the button has been mocked and the `aria-selected` prop is not needed for the component `Tab`, we can remove this line. I think `activeTab && styles.activeTab` is enough. Or you can demonstrate it tomorrow.